src/app/utils/item-utils.ts

```diff
@@ -296,6 +303,55 @@ export function getItemYear(item: DimItem) {
   }
 }
 
+/**
+ * This function indicates whether a mod's stat effect is active on the item.
+ *
+ * For example, powerful friends only gives its stat effect if another arc mod is
+ * slotted or some other item has a charged with light arc mod slotted.
+ * This will return true if another arc mod is slotted or if we can pass in the
+ * other slotted mods via modsOnOtherItems, an arc charged with light mod is found.
+ *
+ * If the plugHash isn't recognised then the default is to return true. This is
+ * because we do not always have access whether a stat is
```

Comment:
Sentence ends abruptly

Reply:
I can't remember what I was going to say there and it probably changed as this function went through a few iterations.